Fix typos

samdotola opened the pull request:
fixed all visible typos in necessary files

khannanov-nil commented on docs/nil/prerequisites.mdx:
```diff
@@ -52,7 +52,7 @@ To install Hardhat:
 ## The =nil; repository
 
-The =nil; cluster, its developer tooling, and additional projects related to =nil; are stored in a monorepo that uses Nix for dependency and project management:
+The =nil; cluster, its developer tooling, and additional projects related to =nil; are stored in a mono repo that uses Nix for dependency and project management:
```
Please change this back to `monorepo` to ensure consistent terminology across other READMEs anddocs. Otherwise, LGTM!

samdotola replied:
Resolved
Changed it back to monorepo
Thanks for the feedback

ukorvl commented:
Let's merge all commits in 1?

khannanov-nil commented:
@samdotola please rebase and squash your commits so that there are no merge commits and there is only one commit for the PR. Thanls!

samdotola commented:
> Let's merge all commits in 1?

Yes
Will do just that 
Thanks

samdotola commented:
> @samdotola please rebase and squash your commits so that there are no merge commits and there is only one commit for the PR. Thanks!

Alright
Thank you

knazarov commented:
If we want to get this change merged, the person (maintainer) who merges needs to create a separate branch, absorb the commit, sign it with their identity, and then get it stamped.

samdotola commented:
> If we want to get this change merged, the person (maintainer) who merges needs to create a separate branch, absorb the commit, sign it with their identity, and then get it stamped. Direct merges are not allowed, because they violate code signing policy.

Thanks for the clarification
I’ve rebased and squashed the commits. I understand the signing policy now. Please let me know if you need anything else from my end.

khannanov-nil commented:
@samdotola here is the PR with the signed commit: https://github.com/NilFoundation/nil/pull/328

In the future, kindly open issues instead of PRs for contributions. This will enable an easier flow for quickly addressing fixes.
